Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilder.cpp

```diff
@@ -14,10 +14,13 @@
 #include <ScriptCanvas/Components/EditorDeprecationData.h>
 #include <ScriptCanvas/Components/EditorGraph.h>
 #include <ScriptCanvas/Components/EditorGraphVariableManagerComponent.h>
+#include <ScriptCanvas/Core/GraphSerialization.h>
 #include <ScriptCanvas/Grammar/AbstractCodeModel.h>
 
 namespace ScriptCanvasBuilderCpp
 {
+    using namespace ScriptCanvas;
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether